hack/local-up-cluster.sh

```diff
@@ -23,7 +23,7 @@ DOCKER_OPTS=${DOCKER_OPTS:-""}
 DOCKER=(docker ${DOCKER_OPTS})
 DOCKERIZE_KUBELET=${DOCKERIZE_KUBELET:-""}
 ALLOW_PRIVILEGED=${ALLOW_PRIVILEGED:-""}
-ALLOW_SECURITY_CONTEXT=${ALLOW_SECURITY_CONTEXT:-""}
+ALLOW_SECURITY_CONTEXT=${ALLOW_SECURITY_CONTEXT:-"true"}
```

Comment:
the way this is used is odd... it doesn't check "true" or "false", just "empty" vs "not empty". defaulting this to "not empty" means to override, you'd have to do `ALLOW_SECURITY_CONTEXT= hack/local-up-cluster.sh`

Would prefer switching to the same pattern as the other admission plugins: `DENY_SECURITY_CONTEXT_ADMISSION` and if non-empty, add the plugin

Reply:
Done! thanks